webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -97,7 +106,8 @@ const ActionsBar = ({
       ouiaId="console-dropdown-item"
       onClick={() => visit(foremanUrl(`/hosts/${hostFriendlyId}/console`))}
       key="console"
-      isDisabled={!isHostActive}
+      isAriaDisabled={!(computeId && isHostActive)}
```

Comment:
what is the difference between `isAriaDisabled` and `isDisabled`?

Reply:
> what is the difference between isAriaDisabled and isDisabled?

`isAriaDisabled` allows the tool tip to be displayed even when the button is greyed out while `isDisabled` turns the tool tip off.